Assets/Mirror/Runtime/Transport/FallbackTransport.cs

```diff
@@ -0,0 +1,137 @@
+// uses the first available transport for server and client.
+// example: to use Apathy if on Windows/Mac/Linux and fall back to Telepathy
+//          otherwise.
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Mirror
+{
+    public class FallbackTransport : Transport
+    {
+        public Transport[] transports;
+
+        // the first transport that is available on this platform
+        Transport available;
+
+        public void Awake()
+        {
+            if (transports == null || transports.Length == 0)
+            {
+                Debug.LogError("FallbackTransport requires at least 1 underlying transport");
```

Comment:
I would really prefer to start using exceptions properly, especially since you have a very similar exception in line 39